### PR TITLE
cmd: Remove not used package in service.tmpl

### LINKED ISSUE #677: Remove not used package in service.tmpl
It looks like we don't need `credential.Provider` anymore, we can remove the import:

```go
"github.com/beyondstorage/go-storage/v4/pkg/credential"
```

## Steps

- [ ] Remove the import from `cmd/definitions/tmpl/service.tmpl`
- [ ] `make build` to trigger generation

### PR DESCRIPTION
**Pull request by RhnSharma**
Fixes #677
Hi @Xuanwo, how does this look?
Let me know if this needs any changes.
Thanks

### CODE REVIEW

File: cmd/definitions/tmpl/service.tmpl [hunk 1]
```diff
@@ -5,7 +5,6 @@ import (
     "context"
     "io"
 
-    "github.com/beyondstorage/go-storage/v4/pkg/credential"
```
Comment (Xuanwo) [inline cmd/definitions/tmpl/service.tmpl:8]: Ooops, it looks like we need to remove `var _ credential.Provider` below too.

Comment (RhnSharma) [reply cmd/definitions/tmpl/service.tmpl:8]: Okay, Do I have to run `make build` again?

Comment (Xuanwo) [reply cmd/definitions/tmpl/service.tmpl:8]: > Okay, Do I have to run `make build` again?

Yes, we need to use `make build` to generate code every time we changed the template.

Comment (RhnSharma) [reply cmd/definitions/tmpl/service.tmpl:8]: I have removed the `var_credential.Provider`. Please take a look at it and let me know if this needs any changes.
Thanks

**Review by Xuanwo — APPROVED** [review]

**Comment by Xuanwo** [comment]
Thanks for your contributions!

**Comment by rbrishabh** [comment]
thanks @RhnSharma! this is amazing <3

**Pull request merged**